### PR TITLE
Fix detection of yield-fixture when combined with parametrization (fixes #991)

### LINKED ISSUE #991: Ordering problem between `slash.fixture` and `slash.parameters.iterate` decorator
There is importance for the order of `slash.fixutre` (in case it is yield fixture) and `slash.parameters.iterate` decorators:
```
import slash


@slash.fixture
@slash.parameters.iterate(my_param=['a', 'b'])
def fixture_1(my_param):
    yield my_param

@slash.parameters.iterate(my_param=['a', 'b'])
@slash.fixture
def fixture_2(my_param):
    yield my_param

def test_a(fixture_1, fixture_2):
    slash.logger.info(f"fixture_1 value: {fixture_1}")
    slash.logger.info(f"fixture_2 value: {fixture_2}")
```

```
slash run -v ~/Documents/tmp/test_me.py
4 tests collected   
[2019-11-07 11:24:13] #1: /home/ayalas/Documents/tmp/test_me.py:test_a(fixture_1.my_param=a,fixture_2.my_param=a)
[2019-11-07 11:24:13] fixture_1 value: <generator object fixture_1 at 0x7efd59951048>
[2019-11-07 11:24:13] fixture_2 value: a
[2019-11-07 11:24:13] #2: /home/ayalas/Documents/tmp/test_me.py:test_a(fixture_1.my_param=a,fixture_2.my_param=b)
[2019-11-07 11:24:13] fixture_1 value: <generator object fixture_1 at 0x7efd597b7b48>
[2019-11-07 11:24:13] fixture_2 value: b
[2019-11-07 11:24:13] #3: /home/ayalas/Documents/tmp/test_me.py:test_a(fixture_1.my_param=b,fixture_2.my_param=a)
[2019-11-07 11:24:13] fixture_1 value: <generator object fixture_1 at 0x7efd597b7d00>
[2019-11-07 11:24:13] fixture_2 value: a
[2019-11-07 11:24:13] #4: /home/ayalas/Documents/tmp/test_me.py:test_a(fixture_1.my_param=b,fixture_2.my_param=b)
[2019-11-07 11:24:13] fixture_1 value: <generator object fixture_1 at 0x7efd59974ba0>
[2019-11-07 11:24:13] fixture_2 value: b
============================================================ Session ended. 4 successful, 0 skipped, 0 failed, 0 erroneous. Total duration: 00:00:00 =============================================================
```

### PR DESCRIPTION
**Pull request by ayalash**

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :bug:`991` Fix detection of yield-fixture when it combained with parameterization
```
Comment (vmalloc) [inline doc/changelog.rst]: "when combined with"

Comment (vmalloc) [reply doc/changelog.rst]: parametrization

File: slash/utils/python.py [hunk 2]
```diff
@@ -41,11 +41,20 @@ def decorator(new_func):
             setattr(decorator, p, orig)
     return decorator
 
+
+def is_generator(func):
```
Comment (vmalloc) [inline slash/utils/python.py]: Why not just use inspect.isgeneratorfunction(get_underlying_func(...)) directly? IIRC it returns the same function if not a generator...

Comment (ayalash) [reply slash/utils/python.py]: You are right... Fixed.

File: doc/changelog.rst [hunk 3]
```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :bug:`991` Fix detection of yield-fixture when combained with parametrization
```
Comment (vmalloc) [inline doc/changelog.rst]: when **combined**

Comment (ayalash) [reply doc/changelog.rst]: Oops :(

**Comment by vmalloc** [comment]
@ayalash it seems that the build is failing

**Comment by ayalash** [comment]
@vmalloc It was a travis problem again :(
After restart everything passed

**Comment by vmalloc** [comment]
Thanks!

**Pull request merged**